src/test/java/org/opensearch/ad/caching/PriorityCacheTests.java

```diff
@@ -36,24 +89,22 @@ public void setUp() throws Exception {
 
         modelManager = mock(ModelManager.class);
 
-        clusterService = mock(ClusterService.class);
-        ClusterSettings settings = new ClusterSettings(
-            Settings.EMPTY,
-            Collections
-                .unmodifiableSet(
-                    new HashSet<>(
-                        Arrays
-                            .asList(
-                                AnomalyDetectorSettings.DEDICATED_CACHE_SIZE,
-                                AnomalyDetectorSettings.MODEL_MAX_SIZE_PERCENTAGE,
-                                AnomalyDetectorSettings.MODEL_MAX_SIZE_PERCENTAGE,
-                                AnomalyDetectorSettings.CHECKPOINT_TTL,
-                                AnomalyDetectorSettings.CHECKPOINT_SAVING_FREQ
```

Comment:
Curious why these settings are not tested in the new changes?

Reply:
These settings were added to `main` after we created `feature/extensions` [here](https://github.com/opensearch-project/anomaly-detection/pull/620/files#diff-00c99f214d22b6de9c5e592fc22fed35b542a8afc7f6b9d23c93e4efd0c2a5c0R103). Probably this test was fetched from `main` because it doesn't incline with the code changes of `feature/extensions`.

Reply:
These settings on feature/extensions are of the type [Duration](https://github.com/opensearch-project/anomaly-detection/blob/feature/extensions/src/main/java/org/opensearch/ad/settings/AnomalyDetectorSettings.java#L183) whereas based on the above PR the type got changed to [Setting](https://github.com/opensearch-project/anomaly-detection/blob/main/src/main/java/org/opensearch/ad/settings/AnomalyDetectorSettings.java#L196). Changing the type would require to fetch more code changes from `main` which we will be doing anyway with the re-fork.
The reason of the conflict here is the code on feature/extenions was from the time when we forked but probably while commenting the code for these tests, we pulled the code from main for the tests.